### PR TITLE
Add organization flag to chronoctl for orgs

### PR DESCRIPTION
**Pull request by desa**
This flag allows users to have a comma delimited list of organization
IDs that the user should given a role in.

  - [ ] Rebased/mergable

Connect #2744 

### The problem

Previously users couldn't specify organizations that the superadmin user should be added to. This created a situation where super admins could not auth into the application.

### The Solution

Allow users to specify a comma separated list of organization IDs that the user should be a part of.

### CODE REVIEW

File: cmd/chronoctl/add.go [hunk 1]
```diff
@@ -2,17 +2,18 @@ package main
...
-	Provider string  `short:"p" long:"provider" description:"Name of the Auth provider (e.g. google, github, auth0, or generic)"`
-	Scheme   string  `short:"s" long:"scheme" description:"Authentication scheme that matches auth provider (e.g. oauth or ldap)"`
-	//Organizations string `short:"o" long:"orgs" description:"A comma separated list of organizations that the user should be added to"`
+	BoltPath      string  `short:"b" long:"bolt-path" description:"Full path to boltDB file (e.g. './chronograf-v1.db')" env:"BOLT_PATH" default:"chronograf-v1.db"`
+	ID            *uint64 `short:"i" long:"id" description:"Users ID. Must be id for existing user"`
+	Username      string  `short:"n" long:"name" description:"Users name. Must be Oauth-able email address or username"`
+	Provider      string  `short:"p" long:"provider" description:"Name of the Auth provider (e.g. google, github, auth0, or generic)"`
+	Scheme        string  `short:"s" long:"scheme" description:"Authentication scheme that matches auth provider (e.g. oauth or ldap)"`
+	Organizations string  `short:"o" long:"orgs" description:"A comma separated list of organizations that the user should be added to"`
```
Comment (nhaugo) [inline cmd/chronoctl/add.go]: By name or id

Comment (desa) [reply cmd/chronoctl/add.go]: ID, but I can do name if that's easier.

Comment (nhaugo) [reply cmd/chronoctl/add.go]: Does list show the name or the id? The user will just need access to the info in case they are totally locked out.

Comment (desa) [reply cmd/chronoctl/add.go]: list shows the ids

Comment (desa) [reply cmd/chronoctl/add.go]: 1. the work flow I was imagining was look to see all users
2. grab an existing users list of ids
3. modify them
4. add new super admin with the modified list

**Pull request merged**